### ISSUE TITLE
chrome_pass install problem

### ISSUE THREAD

**Issue by grauschnabel** [hsanson/chrome-pass#36]
I tried to use the install manual on this project, but getting into troubles during this step:

```
martin@martin-fedora:~$ chrome_pass install
Traceback (most recent call last):
  File "/usr/lib64/python3.12/importlib/metadata/__init__.py", line 285, in __getitem__
    return next(iter(self.select(name=name)))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
StopIteration

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/martin/.local/bin/chrome_pass", line 8, in <module>
    sys.exit(run())
             ^^^^^
  File "/home/martin/.local/lib/python3.12/site-packages/chrome_pass/__init__.py", line 310, in run
    install(native_path_chrome(), EXTENSION_ID)
  File "/home/martin/.local/lib/python3.12/site-packages/chrome_pass/__init__.py", line 285, in install
    native_app = find_chrome_pass_path()
                 ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/martin/.local/lib/python3.12/site-packages/chrome_pass/__init__.py", line 254, in find_chrome_pass_path
    entry_point = entry_points().select(
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/importlib/metadata/__init__.py", line 287, in __getitem__
    raise KeyError(name)
KeyError: 0
martin@martin-fedora:~$ which chrome_pass 
~/.local/bin/chrome_pass
```

On firefox the pass implementation works just fine, but for chromium it seems to be tricky.  Thanks for your thoughts.

**Comment by hsanson** [comment]
Thanks for reporting... can you tell me what version of Python you are using? Seems there was a breaking change in how `entry_points` work and indexing it as an array breaks on some versions of Python.

I need to figure out what is the proper way to access the entry points and then try to test it on a version of python that presents the issue.

The exact piece of code that breaks seems to be this:

```python
entry_point = entry_points().select(
       name='chrome_pass', group='console_scripts')[0]
```

That `[0]` indexing at the end is the problem since entry_points() seems to be an array but in fact it is not... or at least that is my current understanding.

Reference:

 - https://github.com/python/cpython/issues/88412

**Comment by grauschnabel** [comment]
Hej, der är faktiskt python 3.12 här. Do you have a fix for that?

**Comment by hsanson** [comment]
Should be fixed in v0.5.1 I just published.
